pkg/exchange/okex/stream_test.go

```diff
@@ -0,0 +1,552 @@
+package okex
+
+import (
+	"context"
+	"os"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/c9s/bbgo/pkg/testutil"
+	"github.com/c9s/bbgo/pkg/types"
+)
+
+func getTestClientOrSkip(t *testing.T) *Stream {
+	if b, _ := strconv.ParseBool(os.Getenv("CI")); b {
+		t.Skip("skip test for CI")
+	}
+
+	key, secret, passphrase, ok := testutil.IntegrationTestWithPassphraseConfigured(t, "OKEX")
+	if !ok {
+		t.Skip("Please configure all credentials about OKEX")
+		return nil
+	}
+
+	return NewStream(key, secret, passphrase)
+}
+
+func TestStream(t *testing.T) {
+	// t.Skip()
+	s := getTestClientOrSkip(t)
+
+	// symbols := []string{
+	// 	"BTCUSDT",
+	// 	"ETHUSDT",
+	// 	"DOTUSDT",
+	// 	"ADAUSDT",
+	// 	"AAVEUSDT",
+	// 	"APTUSDT",
+	// 	"ATOMUSDT",
+	// 	"AXSUSDT",
+	// 	"BNBUSDT",
+	// 	"SOLUSDT",
+	// 	"DOGEUSDT",
+	// }
+
+	t.Run("Auth test", func(t *testing.T) {
+		s.Connect(context.Background())
+		c := make(chan struct{})
+		<-c
+	})
+
+	t.Run("book test", func(t *testing.T) {
+		s.Subscribe(types.BookChannel, "BTCUSDT", types.SubscribeOptions{
+			Depth: types.DepthLevel50,
```

Comment:
How much depth support is available?

Reply:
channel                depth
books                   400  
books5                 5
bbo-tbt                1 snapshot
books-l2-tbt       400
books50-l2-tbt  50

Reply:
added, will update